include/dnssnoop.h

```diff
@@ -54,14 +73,12 @@ struct inflight_dns_query {
   pid_t tid;
   // userspace pid, kernel tgid
   pid_t pid;
-  uid_t uid;
-  gid_t gid;
   u64 cgroup_id;
   u64 start_time;
   u16 transaction_id;
   char command[COMMAND_BUF_SIZE];
-  char thread_name[THREAD_NAME_BUF_SIZE];
   char domain_name[DOMAIN_NAME_BUF_SIZE];
+  char resolved_addresses[RESOLVED_ADDRESSES_MAX][IP_BUF_SIZE];
```

Comment:
should we make this an answers struct?

Reply:
I'm not sure what you mean.

If you mean instead of just a list of IPs address we should have a list of Structs containing all the fields in an answer, we're not parsing anything else yet & yeetd doesn't support things like unions, plus we're already massively blowing up the size of the ring buf message, I'm not sure I want to blow them up even more for no reason,